_posts/2024-05-16-Top-3-Scenarios-Where-Managing-Multiple-Clusters-with-One-OpenSearch-Dashboards-is-a-Life-Saver.md

```diff
@@ -0,0 +1,96 @@
+
+Also, the referenced data sources from the current map layers are indicated by clicking on the icon on the top navigation bar.
+
+<img src="/assets/media/blog-images/2024-05-16-top3-scenarios-where-managing-multiple-clusters-with-one-opensearch-dashboards-is-a-life-saver/maps_2.png"/>
+
+## Scenario 2: Centralized data management and governance
+
+In addition to analyzing multiple data sources in a unified dashboard, you can manage remote clusters operationally. You can manage your indexes, security policies, notifications, and ML models and then select the target data source from the unified dashboard.
+
+OpenSearch Dashboards allows you to perform all index operations, such as creating, reading, updating, and deleting indexes, as well as managing index mappings, templates, and aliases, across multiple data sources from a single instance. 
+
+You can switch between different data sources using the upper-right navigation bar, select a specific index, and choose the desired operation. Selecting on an index name will display detailed information about that index. The data source id, which is the unique identifier (UUID) is embedded in the URL, allowing you to bookmark and share specific pages directly. 
+
+The following images show the data source picker in the top right hand corner within Index State Management.
+
+<img src="/assets/media/blog-images/2024-05-16-top3-scenarios-where-managing-multiple-clusters-with-one-opensearch-dashboards-is-a-life-saver/indexes_overview.png"/>
+
+<img src="/assets/media/blog-images/2024-05-16-top3-scenarios-where-managing-multiple-clusters-with-one-opensearch-dashboards-is-a-life-saver/detailed_index.png"/>
+
+OpenSearch Dashboards supports managing security across multiple data sources. Administrators can centrally manage roles and internal users for different clusters by selecting the desired cluster from the DATA SOURCES dropdown menu, as shown in the following image.
+
+<img src="/assets/media/blog-images/2024-05-16-top3-scenarios-where-managing-multiple-clusters-with-one-opensearch-dashboards-is-a-life-saver/roles_overview.png"/>
+
+Users can also manage notification channels, as shown in the following image.
+
+<img src="/assets/media/blog-images/2024-05-16-top3-scenarios-where-managing-multiple-clusters-with-one-opensearch-dashboards-is-a-life-saver/channels_overview.png"/>
+
+Users can manage remote ML models, as shown in the following image. The pre-trained models are deployed to the selected cluster through the Dev Tools console. 
+
+<img src="/assets/media/blog-images/2024-05-16-top3-scenarios-where-managing-multiple-clusters-with-one-opensearch-dashboards-is-a-life-saver/models_overview.png"/>
+
+The Dev Tools console, shown in the following image, is integrated with the multiple data sources feature. Users can select the target data source and execute commands to query ot modify the backend data directly from a central location. 
```

Comment:
In the second sentence, we need to choose either "to query" or "to modify".

Reply:
Right, will remove `to modify`